client/order/orderbook.go

```diff
@@ -437,6 +437,29 @@ func (ob *OrderBook) ResetEpoch() {
 	ob.epochQueue.Reset()
 }
 
+// Reset clears the orderbook and its associated epoch queue. This should be
+// called when a trade suspension does not persist the orderbook.
+func (ob *OrderBook) Reset() {
+	ob.seqMtx.Lock()
+	ob.seq = 0
+	ob.seqMtx.Unlock()
+
+	ob.marketID = ""
+
+	ob.noteQueueMtx.Lock()
+	ob.noteQueue = make([]*cachedOrderNote, 0)
+	ob.noteQueueMtx.Unlock()
+
+	ob.ordersMtx.Lock()
+	ob.orders = make(map[order.OrderID]*Order)
+	ob.ordersMtx.Unlock()
+
+	ob.buys = NewBookSide(descending)
+	ob.sells = NewBookSide(ascending)
+
```

Comment:
Is it ok that marketID, buys, and sells aren't protected by a mutex?

Reply:
Good catch, they all need mutexes due to `Reset`.